advanced-vector-search/README.md

```diff
@@ -57,45 +44,59 @@ We encourage you to try different indexers and different options for other index
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-## Requirements
+For this example you can use one of these 2 datasets [ANN_SIFT10K or ANN_SIFT1M](http://corpus-texmex.irisa.fr/), which are datasets comprised of three vector sets:  
 
-Be sure to create an environment with Python 3.7 or higher installed, then you also need to have the requirements of this example,
-which are `jina`, `docker` and `click`.
+- 10K or 1M to index
+- 100 or 10K to query
+- 25K or 100k to train
 
-In order to run the example with the different indexers, make sure to pull the docker images from the [Jina Hub repository](https://hub.docker.com/u/jinahub)
+We will work with ANN_SIFT10K (siftsmall), that are [SIFT](https://en.wikipedia.org/wiki/Scale-invariant_feature_transform) descriptors for some image dataset. 
+But if you wish you could use a larger datasets from the same [source](http://corpus-texmex.irisa.fr/). 
 
-## Prepare the data
+A [docker image](https://hub.docker.com/r/jinahub/app.example.advancedvectorsearch) is published where the ANN_SIFT1M (sift) has already been indexed
```

Comment:
Not sure why we have these line breaks. They look odd

Reply:
I prefer to have more line breaks because I don't like super long sentences

Reply:
I think we should keep to the standards. Most readmes I've seen don't break in the middle of a paragraph. If the editor is the problem, make sure to enable `Soft wraps` or `Wrap lines` (depends on which one you use)